3-cloud-computing/3.2.3-auto-scaling.md

```diff
@@ -16,21 +16,20 @@ Read more about User Data in the [AWS documentation](https://docs.aws.amazon.com
 
 ## Exercise 1 - Create EC2 instance with User Data
 
-In this exercise we will create an EC2 instance and use a User Data script to provision the instance as host for a Java web application.
+In this exercise we will create an EC2 instance and use a User Data script to provision the instance as host for a Java web application. We recommend configuring a machine to run [Spring's PetClinic Project](https://github.com/spring-projects/spring-petclinic) on [CentOS Stream 9](https://www.centos.org/download/aws-images/). But you are free to do this exercise with another web application and distro.
 
-1. Using your notes from previous sections write a shell script which does the following:
-  1. Install Tomcat and configures it to runs as a service.
-  2. Install Maven.
-  3. Download and build a Java Web application such as Spring Petclinic.
-  4. Deploy your application to Tomcat and make it accessible at http://[PUBLIC_IP]/[APPLICATION_NAME].
+1. Create a script to pass into `--user-data`. In general the script needs to:
+    * Install all dependencies for your web application
```

Comment:
might be useful to explain where to find these dependencies, incase the apprentices have not dealt with a Java app before.